src/index.tsx

```diff
@@ -212,6 +213,29 @@ class App extends React.Component<{}, AppState> {
       }
       this.forceUpdate();
       event.preventDefault();
+      // Copy Styles: Cmd-Shift-C
+    } else if (event.metaKey && event.shiftKey && event.code === "KeyC") {
+      const element = elements.find(el => el.isSelected);
+      if (element) {
```

Comment:
What should the behavior be for more than 1 element?

Reply:
It'd take the first that satisfies the condition `isSelected`.
The approach I took was that we're able to copy styles from a single shape, but when pasting we can do it on multiple shapes.